target/linux/ath79/dts/ar9344_devolo_dlan2-2400-ac.dts

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9344.dtsi"
+
+/ {
+	model = "Devolo Magic 2 Wifi";
+	compatible = "devolo,dlan2-2400-ac", "devolo,magic2wifi", "qca,ar9344";
```

Comment:
You have to decide here: You can only have _one_ name for device definition.
I would go for the actual device name here (devolo,magic-2-wifi): This would then be your _only_ value for the cases in the files above, for the compatible (despite soc), for the Makefile block name, and for the DTS name (with _ instead of ,).